diffpy/pdfmorph/morphs/morph.py

```diff
@@ -47,10 +47,12 @@ class Morph(object):
     Instance attributes:
 
     config      -- dictionary that contains all configuration variables
+    "Objective" is the 1st input file, displayed in red
     xobjin      -- last objective input x data
     yobjin      -- last objective input y data
     xobjout     -- last objective result x data
     yobjout     -- last objective result y data
+    "Reference: is the 2nd input file, displayed in blue
```

Comment:
I think we wanted to move away from the "reference" and "objective" names as this is confusing.  Which is the target curve and which is the morphed one?   Maybe use "target" and "morphed" which seems to be much less ambiguous.

To fix this properly we also want to refactor the variable names, so `xobjin` would go to `xtargetin` and `xrefin` would go to ` xmorphedin`